src/user/models/user.entity.ts

```diff
@@ -89,12 +89,14 @@ export class UserEntity {
   })
   donations: DonationEntity[];
 
+  @Exclude()
```

Comment:
Why Exclude?

Reply:
This will only exclude the data in responses, the relationship will continue to exist.
I've excluded social media posts and invited users because we aren't supporting it at the moment but I'm okay with leaving it as it was if still requested.